_gsocproposals/2019/proposal_ActsGpuRace.md

```diff
@@ -6,6 +6,7 @@ year: 2019
 organization: 
  - LAL
  - MdS
+ - CERN
```

Comment:
since Lauren is from Stanford, add:
  - Stanford University

Reply:
Yes. Done.